### ISSUE TITLE
Version incompatibility when upgrading pipeline to 1.1.0

### ISSUE THREAD

**Issue by Enarior** [SigmaHQ/pySigma-pipeline-windows#3]
Hello,

I tried upgrading pipeline to latest version after 4688 events were added, but I get a compatibility error.
`pip install pysigma-pipeline-windows --upgrade`
```
Installing collected packages: pysigma, pysigma-pipeline-windows
  Attempting uninstall: pysigma
    Found existing installation: pysigma 0.8.12
    Uninstalling pysigma-0.8.12:
      Successfully uninstalled pysigma-0.8.12
  Attempting uninstall: pysigma-pipeline-windows
    Found existing installation: pysigma-pipeline-windows 1.0.1
    Uninstalling pysigma-pipeline-windows-1.0.1:
      Successfully uninstalled pysigma-pipeline-windows-1.0.1
ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
sigma-cli 0.5.3 requires pysigma<0.9.0,>=0.8.4, but you have pysigma 0.9.1 which is incompatible.
pysigma-pipeline-sysmon 1.0.1 requires pysigma<0.9.0,>=0.8.1, but you have pysigma 0.9.1 which is incompatible.
pysigma-pipeline-crowdstrike 0.1.8 requires pysigma<0.9.0,>=0.8.1, but you have pysigma 0.9.1 which is incompatible.
pysigma-backend-splunk 0.3.7 requires pysigma<0.9.0,>=0.8.1, but you have pysigma 0.9.1 which is incompatible.
pysigma-backend-qradar 0.2.1 requires pysigma<0.9.0,>=0.8.1, but you have pysigma 0.9.1 which is incompatible.
pysigma-backend-opensearch 0.1.4 requires pysigma<0.9.0,>=0.8.12, but you have pysigma 0.9.1 which is incompatible.
pysigma-backend-insightidr 0.1.8 requires pysigma<0.9.0,>=0.8.1, but you have pysigma 0.9.1 which is incompatible.
pysigma-backend-elasticsearch 0.1.2 requires pysigma<0.9.0,>=0.8.12, but you have pysigma 0.9.1 which is incompatible.
Successfully installed pysigma-0.9.1 pysigma-pipeline-windows-1.1.0
```

To reproduce, install sigma-cli from pip and upgrade pysigma-pipeline-windows using pip as well.

I tried `pip install --force-reinstall -v "pysigma==0.8.12"`, which gives me : 
```ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behavpendency conflicts.
pysigma-pipeline-windows 1.1.0 requires pysigma<0.10.0,>=0.9.1, but you have pysigma 0.8.12 which is incompatible.
```
With both versions I indeed get errors when using `sigma convert`.

Am I missing something on how to upgrade and how versions of the different components of sigma-cli interact, is this version meant to work with a later version of sigma-cli ?

Thanks !

**Comment by thomaspatzke** [comment]
Sorry for the inconvenience, but pysigma-pipeline-windows 1.1.0 is indeed only compatible with pySigma 0.9 while the current version of pySigma is still on 0.8 and therefore not compatible. The pipeline also utilizes a breaking change of pySigma 0.9 and therefore it's not just a version number incompatibility.

I'm currently wotking on the next Sigma CLI release that updates to pySigma 0.9.

**Comment by Enarior** [comment]
Thank you for your answer.
I'll keep the changes of pipeline 1.1.0 in a custom pipeline as a YAML file which works fine for now, and upgrade when new sigma-cli is released.

Thank you again for the quick answer, and for your awesome work on sigma-* !

**Comment by thomaspatzke** [comment]
I've released [sigma-cli 0.6.0](https://github.com/SigmaHQ/sigma-cli/releases/tag/v0.6.0) that includes pySigma 0.9 and plugin support this morning. If you use the Splunk backend, you can switch to it. Further backends will be migrated in the next time.

**Comment by Enarior** [comment]
Works like a charm !
Thanks.
